docs/src/configuration.md

```diff
@@ -69,6 +69,37 @@ and the user-provided one are used in the user configuration if
 `stylix.base16Scheme` is not changed in the user config. If that is the case,
 only the user override is used.
 
+### Extending
+
+When passing colors to unsupported targets, it is possible to access values from the configured
+color scheme through `lib.stylix.colors`. It contains an attrset, populated by
+[base16.nix](https://github.com/SenchoPens/base16.nix)'s `mkSchemeAttrs`.
```

Comment:
```suggestion
color scheme through `lib.stylix.colors`.
```

This sentence seems unnecessary since we link to `mkSchemeAttrs` below.

Reply:
I think it's better to state the origin of the value explicitly, as in "generated by", after the code example